batchlib/analysis/pixel_level_analysis.py

```diff
@@ -0,0 +1,192 @@
+import matplotlib
+matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+# import seaborn as sns
+from matplotlib.pyplot import imshow
+
+import pathlib
+import argparse
+import os
+import os.path
+from concurrent import futures
+from glob import glob
+import json
+import h5py
+import numpy as np
+from tqdm import tqdm
+from time import sleep
+from ..base import BatchJob, BatchJobWithSubfolder
+from ..util.plate_visualizations import well_plot
+
+
+def open_hdf5(filename, *args, **kwargs):
```

Comment:
Please use `util.open_file` instead. It implements the same logic, but avoids infinite loops and it also supports opening n5 files.